etc/ReplayOfflineConfiguration.py

```diff
@@ -35,7 +35,10 @@
 setConfigVersion(tier0Config, "replace with real version")
 
 # Set run number to replay
-# 367102 - Collisions 2023 - 1200b - 0.5h long - all components IN
+# 369998 - Collisions 2023 - 1800b - 1h long - ALL components in
+# 365537 - Splashes 2023 - 2h40 long - CSC, CTPPS, CTPPS_TOT, DT, GEM, PIXEL, RPC, TRACKER OUT
+# 375832 - Cosmics 3.8T 2023 - 3h long - CTPPS, CTPPS_TOT OUT
+
 setInjectRuns(tier0Config, [369998, 365537, 375832])
```

Comment:
@GermanGiraldo @LinaresToine IIUC 
checking https://twiki.cern.ch/twiki/bin/view/CMSPublic/CompOpsTier0TeamReplayGoodRuns
we don't have any 900GeV run to be used for replays from past years, right?
Also I couldn't find information about 375832 in it, would it be possible to add?

Reply:
You are right. We don't have 900GeV runs. I'll update the table with run 375832